<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDeviceManager</class>
 <widget class="QDialog" name="DlgDeviceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgDeviceManager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Device List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QPushButton" name="RemoveDevButton">
        <property name="text">
         <string>Remove Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QTableWidget" name="DeviceTable">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="OkButton">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add Device</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="NewDevType"/>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="AddDevButton">
        <property name="text">
         <string>Add Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QTextEdit" name="NewDevName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Device Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Device Num</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignLeft">
       <widget class="QSpinBox" name="NewDevNum"/>
      </item>
      <item row="3" column="3" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
